### PR TITLE
Add set/get bulk functions typedefs for oid objects

### PR DESCRIPTION
**Pull request by kcudnik**

### CODE REVIEW

File: inc/saitypes.h [hunk 1]
```diff
@@ -1128,6 +1128,51 @@ typedef sai_status_t (*sai_bulk_object_remove_fn)(
...
+        _In_ uint32_t object_count,
+        _In_ const sai_object_id_t *object_id,
+        _In_ const sai_attribute_t *attr_list,
+        _In_ sai_bulk_op_error_mode_t mode,
+        _Out_ sai_status_t *object_statuses);
+
+/**
+ * @brief Bulk objects get attributes.
+ *
+ * @param[in] object_count Number of objects to set on attribute
```
Comment (atitjain) [inline inc/saitypes.h]: set -> get

File: inc/saitypes.h [hunk 2]
```diff
@@ -1128,6 +1128,51 @@ typedef sai_status_t (*sai_bulk_object_remove_fn)(
...
+ * @param[in] object_id List of object ids
+ * @param[in] attr_count List of attr_count. Caller passes the number
+ *    of attribute for each object to create.
```
Comment (atitjain) [inline inc/saitypes.h]: create -> get

File: inc/saitypes.h [hunk 3]
```diff
@@ -1128,6 +1128,51 @@ typedef sai_status_t (*sai_bulk_object_remove_fn)(
         _In_ sai_bulk_op_error_mode_t mode,
         _Out_ sai_status_t *object_statuses);
 
+/**
+ * @brief Bulk objects set attributes.
+ *
+ * @param[in] object_count Number of objects to set on attribute
+ * @param[in] object_id List of object ids
+ * @param[in] attr_list List of attributes for every object, one per object.
```
Comment (atitjain) [inline inc/saitypes.h]: Does this allow a different attribute to be set on different objects in the list?

Comment (kcudnik) [reply inc/saitypes.h]: yes

**Pull request merged**